agent/tests/test_destination.py

```diff
@@ -27,7 +27,8 @@ def test_destination(cli_runner):
 
 def test_edit_destination(cli_runner):
     prev_dest = HttpDestination().load()
-    result = cli_runner.invoke(agent_cli.destination, input='y\ntoken\n\ny\nhttp://squid:3128\n\n\n')
+    result = cli_runner.invoke(agent_cli.destination, input='\ntoken\ny\nhttp://squid:3128\n\n\n')
```

Comment:
set ANODOT_API_URL env var as `http://app.anodot.com` and here put `http://dummy_destination` to test the new functionality